kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-common/src/main/java/org/kie/server/remote/rest/common/util/RestUtils.java

```diff
@@ -118,6 +122,28 @@ public static String getContentType(HttpHeaders headers) {
         return contentType;
     }
 
+    public static MediaType getMediaType(HttpHeaders httpHeaders) {
```

Comment:
it is, and it's duplicated to avoid additional dependency (kie-remote-common) just for one or two methods as it would bring quite few transitive deps